src/lib.rs

```diff
@@ -1,3 +1,60 @@
+//! RON is a simple config format which looks similar to Rust syntax.
```

Comment:
I think having a block comment aka `/*! ... !*/` would be more convenient for this block